Helper/Data.php

```diff
@@ -202,4 +202,24 @@ public function setOrderStatus(int $order_id, string $status)
 
         return;
     }
+
+    /**
+     * Get the correct weight type
+     *
+     * @param string|null $weight
+     *
+     * @return int
+     */
+    public function getWeightTypeOfOption(?string $weight): int
+    {
+        $weightType = $this->getGeneralConfig(
+            'print/weight_indication'
+        );
+
+        if ($weightType === 'kilo') {
+            return (int) ($weight * 1000);
+        }
+
+        return (int) $weight ?: 1000;
```

Comment:
Wat is hier die 1000? Een default waarde? Misschien even een constante van maken.

Reply:
done